### PR TITLE
Media stack Mid-Year review

### PR DESCRIPTION
**Pull request by ferjm**
This change is [Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/180)

### CODE REVIEW

File: _posts/2019-07-03-media-update-h1-2019.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+The numbers we are getting are already pretty nice. You might not be able to see it clearly on the video, but the renderer CPU time for the non hardware accelerated playback is ~8ms, compared to the ~1ms of CPU time that we get with the accelerated version.
+
+#### Improved web compatibility of our media elements implementation
+
+We also got a bunch of other smaller features that significantly improved the web compatibility of our media elements.
+
+- [ferjm](https://github.com/servo/ferjm) [added](https://github.com/servo/servo/pull/22399) support for the HTMLMediaElement `poster` frame attribute
+- [swarnimarun](https://github.com/swarnimarun) [implemented](https://github.com/servo/servo/pull/23236) support for the HTMLMediaElement `loop` attribute
+- [jackxbritton](https://github.com/jackxbritton) [implemented](https://github.com/lucasfantacuci) the HTMLMediaElement `crossorigin` attribute logic.
```
Comment (jdm) [inline _posts/2019-07-03-media-update-h1-2019.md]: Wrong link for the PR?

**Review by jdm — APPROVED** [review]

**Review by Manishearth — APPROVED** [review]

**Comment by ferjm** [comment]
Oh, no bors-servo here.

**Pull request merged**